### ISSUE TITLE
Support streaming decoder

### ISSUE THREAD

**Issue by krzyzacy** [ghodss/yaml#31]
aka https://github.com/go-yaml/yaml/pull/301

**Comment by krzyzacy** [comment]
@luxas in the meantime we already have https://github.com/kubernetes/apimachinery/blob/master/pkg/util/yaml/decoder.go

**Comment by luxas** [comment]
@krzyzacy Yup, saw that from your other comments in the linked thread and started using it, so I'm good at the moment as well. But maybe this anyway should be upstreamed...
